demo/docbot/ingestion.py

```diff
@@ -8,7 +9,22 @@
 # check out the _bulk python client helper.
 
 # Documents should be ingested from the /data/ dir
+DATA_PATH = os.path.abspath(os.path.join(__file__,  "..", "..","..","data"))
+
+def read_files_from_data() -> list:
+  result = []
+
+  for f in os.listdir(DATA_PATH):
```

Comment:
One other thing, this fails silently. As much as raising exceptions isn't fun if it cant find the file the rest of our project will fail because this will "pass" but there wont be any data. Lets add a check for the file and throw a file not found exception if there are no files with .json extensions.